superset-frontend/src/components/ReportModal/HeaderReportActionsDropdown/index.tsx

```diff
@@ -46,21 +47,17 @@ export default function HeaderReportActionsDropDown({
   chart?: ChartState;
 }) {
   const dispatch = useDispatch();
-  const report: AlertObject = useSelector<any, AlertObject>(state => {
+
+  const findReport = useSelector<any, AlertObject>(state => {
     if (dashboardId) {
       return state.reports.dashboards?.[dashboardId];
     }
     if (chart?.id) {
-      return state.reports.charts?.[chart.id];
+      return state.reports.charts?.[chart?.id];
     }
     return {};
   });
-  // const report: ReportObject = Object.values(reports).filter(report => {
-  //   if (dashboardId) {
-  //     return report.dashboards?.[dashboardId];
-  //   }
-  //   // return report.charts?.[chart?.id]
-  // })[0];
+  const report = findReport;
```

Comment:
just set the `useSelector` code snippet as the report

Reply:
I don't fully understand what you mean here

Reply:
If `report = findReport` just use `findReport` wherever you'd use `report`.